### ISSUE TITLE
Unexpected callable error in no-unsanitized

### ISSUE THREAD

**Issue by rpl** [mozilla/addons-linter#3202]
### Describe the problem and steps to reproduce it:

As originally reported in mozilla/eslint-plugin-no-unsanitized#124, it is possible for some minified code included in an extension, e.g. as the following one included in "Vimium C":

```
(e = this.n[n.i])(i, r)
```

to trigger an "Unexpected callable" error while the minified code is being processed by the rules provided by the eslint-plugin-no-unsanitized plugin.

### What happened?

A linting error with message "Unexpected callable: in no-unsanitized" is preventing the addon to be signed.

### What did you expect to happen?

eslint-plugin-no-unsanitized should be able to handle the particular AST that is currently triggering the linting error.

### Anything else we should know?

The issue has to be fixed on the eslint-plugin-no-unsanitized side, mozilla/eslint-plugin-no-unsanitized#124 is the tracking issue for it.

This issue is meant to track its resolution on the addons-linter side (once the bug is fixed on the eslint plugin side, the dependency should be updated in the addons-linter repo)

**Comment by rpl** [comment]
Closing as fixed (by #3205)

**Comment by denishowew** [comment]
I still get this error from `web-ext lint` (5.0.0) on some large js files.

**Comment by rpl** [comment]
@denishowew is the error explicitly mentioning the about no-unsanitized plugin?

with recent versions of the addons-linter there is an issue related to linting large js files bundles (triggered by upgrading the eslint dependency to eslint v7, an existing issue related to this is #3244). 

Locally you may workaround that issue by running node with an higher max heap size value (e.g. by setting the node options using an environment var NODE_OPTIONS=--max-old-space-size=4096`)

would you mind to double-check if you issue is actually related to #3244?

**Comment by denishowew** [comment]
Sorry, it is no-unsanitized but mine is "Unexpected Callee" not "callable".  Probably a different bug.
